charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       - name: gcp-cloud-controller-manager
         image: {{ index .Values.images "cloud-controller-manager" }}
         imagePullPolicy: IfNotPresent
+        {{- if le (semver .Values.kubernetesVersion | (semver "1.31.0").Compare) 0 }}
+        command: ["/cloud-controller-manager"]
+        {{- end}}
```

Comment:
Not entirely sure whether that is the way to go. This would work, but it is a bit unwieldy and we should remove the condition once older versions drop out of support.

Since our image uses `CMD` and the upstream one uses `ENTRYPOINT` to specify the binary to run during the build process, we could just use `command` for both. I suppose that would restart all CCMs if we update. Not sure if that is actually a problem.